Sources/NIOHTTPCompression/HTTPResponseDecompressor.swift

```diff
@@ -16,12 +16,7 @@ import CNIOExtrasZlib
 import NIO
 import NIOHTTP1
 
-public final class NIOHTTPResponseDecompressor: ChannelDuplexHandler, RemovableChannelHandler {
-    public typealias InboundIn = HTTPClientResponsePart
-    public typealias InboundOut = HTTPClientResponsePart
-    public typealias OutboundIn = HTTPClientRequestPart
-    public typealias OutboundOut = HTTPClientRequestPart
-
+public enum NIOHTTPDecompression {
```

Comment:
could we move this to a separate file? Maybe `HTTPDecompression.swift`?

Reply:
done!